walkthroughs/week-3-libraries/datastore/datastore-walkthrough.md

```diff
@@ -153,6 +153,11 @@ Modify your servlet file to take the data entered by users in your "contact me"
 form and store it as an entity in Datastore. Get this working before
 you worry about loading the data.
 
+The first time you deploy your application, you may encounter an error like this when
```

Comment:
Are these your changes? I'm not sure they should be included.

Reply:
No they are not not sure where thy came from!

Reply:
Ok, then I think you can go ahead and leave them out of the PR. Run a git command to restore those files.

Reply:
i just checked the original git files they have those lines as well i don't know why they are showing them as an addition.

Reply:
so after i do suggest fixes i create a new pull request?